### PR TITLE
Add printout to every 10th print

### PR DESCRIPTION
**Pull request by JanneKiiskila**
Add a starting print and print for every 10 blinks, so that you can
monitor in a CI job if the app is running normally.

### CODE REVIEW

File: mbed_app.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "target_overrides": {
+        "*": {
+            "platform.stdio-baud-rate": 1152000,
```
Comment (0xc0170) [inline mbed_app.json]: I think our examples still use 9600 by default?

File: main.cpp [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
 DigitalOut led1(LED1);
 
 // main() runs in its own thread in the OS
 int main() {
+    uint32_t ctr = 0;
+
+    printf("Starting mbed-os-example-blinky\n");
     while (true) {
+        ctr++;
+        if (ctr % 10 == 0) {
```
Comment (0xc0170) [inline main.cpp]: I understand that there is a board without LED, but then blinky becomes more than just a blinky, as it needs to now explain how to setup serial communication, and what it prints.

Not rather to explain if the board does not have app LED, a user needs to add external LED for this example to be functional?

File: mbed_app.json (hunk 1 above)
Comment (JanneKiiskila) [reply mbed_app.json]: Oh yes, flashback from 1992 back to the years I used to use a  modem. :-)

File: main.cpp (hunk 2 above)
Comment (JanneKiiskila) [reply main.cpp]: Well, then one has to add those instructions on a board per board basis, as there is no "common way of doing it", I think. RTL8195AM should get those instructions yes.

But, the CI need still exists.

**Comment by JanneKiiskila** [comment]
Well, if someone doesn't want the serial - it's up to them. It can be considered an optional part of it. Nevertheless, anyone becoming more serious - will need to set up the serial sooner or later. There are instructions available for it, we should just link to them.

**Comment by JanneKiiskila** [comment]
@tung7970  @Archcady  - can you share the steps you've used to connect the LED? To which pins exactly do you connect it and what type of LED do you recommend?

**Pull request closed without merging**